helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -586,6 +586,26 @@ processors:
         new_name: container.memory.usage
   {{- end }}
 
+  {{- if or .Values.autodetect.prometheus .Values.autodetect.istio }}
+  # This processor is used to remove excessive istio attributes to avoid running into the dimensions limit.
+  # This configuration assumes single cluster istio deployment. If you run istio in multi-cluster scenarios,
+  # you may need to adjust this configuration.
+  attributes/istio:
+    include:
+      match_type: regexp
+      metric_names:
+        - istio_.*
+    actions:
+      - action: delete
+        key: source_cluster
+      - action: delete
+        key: destination_cluster
+      - action: delete
+        key: source_canonical_service
```

Comment:
If we are removing the canonical_service then we might as well remove the canonical_revision labels

Reply:
Makes sense. Removed `source_canonical_revision` and `destination_canonical_revision`